examples/src/keyrings/aws_kms_mrk_keyring_example.py

```diff
@@ -0,0 +1,151 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example sets up the KMS MRK (multi-region key) Keyring
+
+KMS MRK keyring interacts with AWS Key Management Service (AWS KMS) to create, encrypt,
+and decrypt data keys using AWS KMS defined Customer Master Keys (CMKs).
```

Comment:
I'm now remembering we want to avoid using "CMK" if possible
Also avoid "MRK keyring" since it's a little redundant; try this out

```suggestion
The AWS Key Management Service (AWS KMS) MRK keyring interacts with AWS KMS to create, encrypt,
and decrypt data keys with multi-region AWS KMS keys (MRKs).
```